src/raftstore/store/store.rs

```diff
@@ -2374,6 +2374,11 @@ impl<T: Transport, C: PdClient> Store<T, C> {
     }
 
     fn on_compact_check_tick(&mut self, event_loop: &mut EventLoop<Self>) {
+        if rocksdb::auto_compactions_is_disabled(&self.kv_engine) {
+            debug!("skip compact check when disabled auto compactions.");
+            return;
```

Comment:
Need to call `register_compact_check_tick` here.

Reply:
How about the `auto_compaction_is_disabled` is false when checking, but is true after trigger a compaction?

Reply:
That's fine, we just want to avoid unnecessary compactions as long as possible.